cmd/app.go

```diff
@@ -1104,12 +1103,18 @@ func processAnswers(
 			answers[parts[0]] = parts[1]
 		}
 	}
-
-	if interactive {
-		// answers to questions will be added to map
-		err := askQuestions(tv, answers)
-		if err != nil {
-			return answers, err
+	// checks if answers or values were passed in before prompting with questions or setting question defaults
+	if ctx.String("answers") == "" && ctx.String("values") == "" {
```

Comment:
Question: do we want to skip this just because some answers are provided? Is it possible that a values in values.yaml is not covered by questions.yaml?